paper.bib

```diff
@@ -19063,6 +19063,151 @@ @Article{Thrampoulidis2015LASSO
+  Author                   = {Shimodaira, A.},
+  Journal                  = {Journal of Statistical Planning and Inference},
+  Year                     = {2000},
+  Number                   = {2},
+  Pages                    = {227 - 244},
+  Volume                   = {90},
+
+  Doi                      = {https://doi.org/10.1016/S0378-3758(00)00115-4},
+  ISSN                     = {0378-3758},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0378375800001154}
+}
+
+@Article{Sugiyama2008,
+  Title                    = {Direct Importance Estimation for Covariate Shift Adaptation},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Nakajima, S. and Kashima, H. and von B\"unau, P. and Kawanabe, M.},
+  Journal                  = {Annals of the Institute of Statistical Mathematics},
+  Year                     = {2008},
+  Number                   = {4},
+  Pages                    = {699--746},
+  Volume                   = {60}
+}
+
+@Article{SUGIYAMA2011735,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {M. Sugiyama and T. Suzuki and Y. Itoh and T. Kanamori and M. Kimura},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24},
+
+  Doi                      = {https://doi.org/10.1016/j.neunet.2011.04.003},
+  ISSN                     = {0893-6080},
+  Keywords                 = {Two-sample test, Homogeneity test, Density ratio estimation, Unconstrained least-squares importance fitting, Pearson divergence},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0893608011001262}
+}
+
+@Article{Sugiyama2011,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Itoh, Y. and Kanamori T. and Kimura M.},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24}
+}
+
+@Book{Sugiyama2012,
+  Title                    = {Density Ratio Estimation in Machine Learning},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Kanamori, T.},
+  Publisher                = {Cambridge University Press},
+  Year                     = {2012},
+
+  Location                 = {Cambridge, UK}
+}
+
+@Article{sugiyama2007covariate,
+  Title                    = {Covariate shift adaptation by importance weighted cross validation},
+  Author                   = {Sugiyama, Masashi and Krauledat, Matthias and M{\~A}{\v{z}}ller, Klaus-Robert},
+  Journal                  = {Journal of Machine Learning Research},
+  Year                     = {2007},
+  Number                   = {May},
+  Pages                    = {985--1005},
+  Volume                   = {8}
+}
+
+@Book{Vapnik1998,
```

Comment:
@Book{Vapnik1998Statistical,
  title      = {Statistical learning theory},
  publisher  = {John Wiley \& Sons, Inc., New York},
  year       = {1998},
  author     = {Vapnik, Vladimir N.},
  series     = {Adaptive and Learning Systems for Signal Processing, Communications, and Control},
  isbn       = {0-471-03003-1},
  note       = {A Wiley-Interscience Publication},
  mrclass    = {62G07 (68T05)},
  mrnumber   = {1641250},
  mrreviewer = {Ilya S. Molchanov},
  pages      = {xxvi+736},
}